.github/workflows/lint.yml

```diff
@@ -1,7 +1,13 @@
 ---
 name: Lint
 
-on: [push]
+on:
+  push:
+    branches:
+      - dev
+  pull_request:
+    branches:
+      - dev
```

Comment:
+ main

Reply:
dev + main
All the jobs (test, lint, security) need to run when a PR is created (for both main and dev)

Reply:
done :)